### ISSUE TITLE
provide docker image for v2 beta

### ISSUE THREAD

**Issue by hex2a** [scaleway/scaleway-cli#781]
please provide a docker image for beta releases

**Comment by loicbourgois** [comment]
Hi @hex2a,

Thank you for the suggestion. 
The CLI V1 is already available on our docker hub: see https://hub.docker.com/r/scaleway/cli.

We plan to do the same for the CLI V2. 

This should be addressed in the next release: see [v2.0.0-beta.2](https://github.com/scaleway/scaleway-cli/milestone/10). 

Let us know if you have other ideas.

Cheers,

**Comment by jerome-quere** [comment]
Hi @hex2a the v2.0.0-beta.2 release in available on docker hub.
```
> ~ docker run scaleway/cli:v2.0.0-beta.2 version
version     2.0.0-beta.2
build-date  2020-03-25T05:00:28PM
go-version  go1.14.1
git-branch  HEAD
git-commit  -
go-arch     amd64
go-os       linux
```
